README.md

```diff
@@ -37,7 +37,7 @@ With this, you can build truly real-time applications like [Leaderboard](https:/
 The easiest way to get started with DiceDB is using [Docker](https://www.docker.com/) by running the following command.
 
 ```bash
-docker run -p 7379:7379 dicedb/dicedb
+docker run -p 7379:7379 dicedb/dicedb --enable-multithreading --enable-watch
```

Comment:
Section below `### Multi-Threading Mode (Experimental)` also needs to be updated? Seems redundant or we can remove the `--enable-watch` flag from below.